assets/scss/modules/_navigation.scss

```diff
@@ -147,3 +145,145 @@ a.back-to-top-link {
   }
 
 }
+
+/*
+Page Navigation
+
+A page navigation element with a previous and next link.
+
+Markup:
+<nav class="page-nav" role="navigation" aria-label="Page navigation">
+  <ul>
+    <li class="page-nav__item page-nav__item--previous">
+      <a class="page-nav__link page-nav__link-previous" title="Navigate to previous part" rel="prev" href="#">
+        <span class="page-nav__label">Previous </span>
+        <span class="page-nav__title">Overview</span>
+      </a>
+    </li>
+    <li class="page-nav__item page-nav__item--next">
+      <a class="page-nav__link page-nav__link-next" title="Navigate to next part" rel="next" href="#">
+        <span class="page-nav__label">Next</span>
+        <span class="page-nav__title">Inheritance Tax reliefs</span>
+      </a>
+    </li>
+  </ul>
+</nav>
+
+Styleguide Navigation.Page Navigation
+*/
+.page-nav {
+  display:block;
+  margin:em(64) 0 0 0;
+  padding:em(12) 0;
+  border-bottom:1px solid $light-grey;
+
+  @include contain-floats;
+
+  @include media(mobile) {
+    margin:em(32) 0 0 0
+  }
+}
+
+.page-nav__item {
+  @include core-19;
+  line-height:1.25;
+  float:left;
+  text-align:right;
+  margin:em(5) 0;
+  padding:0;
+  width:49%;
+
+  @include media(mobile) {
+    @include core-14;
+    line-height:1.14286;
+    float:none;
+    width:100%
+  }
+}
+
+.page-nav__item--previous {
+  float:left;
+  text-align:left;
+}
+
+.page-nav__link {
+  display:block;
+  color:$govuk-blue-colour;
+  text-decoration:none;
+
+  &:hover, &:active {
+    background-color:$shaded-bg-grey;
+  }
+
+  &:focus {
+    outline: em(3) solid $yellow;
```

Comment:
do you want this to be an `em` value? usually outlines and borders are `px` as we don't wish them to grow when a user zooms or has a different `font-size`